build/templates/_grpc_stub_interpreter.py/initialization_method.py.mako

```diff
@@ -15,9 +15,13 @@
 %>\
 
     def ${full_func_name}(${method_decl_params}):  # noqa: N802
+        metadata = (
+            ('x-api-key', self._grpc_options.api_key),
```

Comment:
FYI, MeasurementLink is using NI-prefixed metadata headers instead of reusing standard(ish) headers: https://dev.azure.com/ni/DevCentral/_git/ASW?path=/Source/Protos/ni/measurementlink/monitoring/v1/metadata.md

Reasoning: "In the future, if we incorporate off-the-shelf HTTP proxy or load balancing servers, these may use the standard HTTP headers differently."

This reasoning may not apply here, so this is just an FYI.

Reply:
The invoke method is the only call site, so it doesn't seem to add much benefit.

Reply:
I don't think the _reasoning_ applies here, as this is a non-standard metadata header already. It could theoretically be (or become) a _common_ one, though. I'll see what Zach has to say about it.

Reply:
Sounds like Zach is happy with whatever. Brad, do you have a particular proposal for how to apply an NI prefix here? To be consistent with grpc-device's naming of [ni-error](https://github.com/ni/grpc-device/blob/bb15a2e7ea2549a7e4ab6fe55049fda7b550250f/source/server/converters.h#L327) trailing metdata, it should probably be ni-api-key. But grpc-device's ni-error metadata hasn't released yet, so we could change that, too, if need be.

Reply:
`ni-api-key` sounds fine to me.

Reply:
Renamed.